### PR TITLE
FIX: open deeplink on iOS

### PR DESCRIPTION
**Pull request by cmgustavo**

### CODE REVIEW

File: ios/BitPayApp/AppDelegate.m [hunk 1]
```diff
@@ -110,6 +112,8 @@ - (BOOL)application:(UIApplication *)application
   openURL:(NSURL *)url
   options:(NSDictionary *)options
 {
+  [RCTLinkingManager application:application openURL:url options:options];
+  [AnalyticsReactNative trackDeepLink:url withOptions:options];
```
Comment (JohnathanWhite) [inline ios/BitPayApp/AppDelegate.m:116]: same for AnaylticsReactNative

File: ios/BitPayApp/AppDelegate.m [hunk 2]
```diff
@@ -14,6 +14,8 @@
 #import <FlipperKitNetworkPlugin/FlipperKitNetworkPlugin.h>
 #import <SKIOSNetworkPlugin/SKIOSNetworkAdapter.h>
 #import <FlipperKitReactPlugin/FlipperKitReactPlugin.h>
+#import <segment_analytics_react_native-Swift.h>
```
[the remark this answers is not in the document]
Comment (cmgustavo) [reply ios/BitPayApp/AppDelegate.m]: Done!

File: ios/BitPayApp/AppDelegate.m (hunk 1 above)
Comment (cmgustavo) [reply ios/BitPayApp/AppDelegate.m]: Done!

**Comment by cmgustavo** [comment]
Just removed all unnecessary `#import`.
`RCTLinkingManager` is needed by React Native to handle deeplinks: https://reactnative.dev/docs/0.67/linking

**Review by JohnathanWhite — APPROVED** [review]

**Pull request merged**